app/scripts/modules/amazon/src/instance/details/instance.details.controller.js

```diff
@@ -99,6 +101,18 @@ module.exports = angular.module('spinnaker.amazon.instance.details.controller',
                 vpcId = loadBalancer.vpcId;
                 return true;
               }
+            }) ||
+            loadBalancer.targetGroups.some(function (targetGroup) {
+              return targetGroup.instances.some(function (possibleInstance) {
```

Comment:
Yeah, but I didn't want it to feel left out with all the rest of `function()`s. Refactor of this class will come later.